stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,929 @@
+						<div class="fd-tokenizer fd-tokenizer--readonly">
+							<div class="fd-tokenizer__inner">
+								<span class="fd-token fd-token--readonly" role="button">
+									<span class="fd-token__text">
+										Bibendum
+									</span>
+								</span>
+								<span class="fd-token fd-token--readonly" role="button">
+									<span class="fd-token__text">
+										Lorem
+									</span>
+								</span>
+								<span class="fd-token fd-token--readonly" role="button">
+									<span class="fd-token__text">
+										Dolor
+									</span>
+								</span>
+								<span class="fd-token fd-token--readonly" role="button">
+									<span class="fd-token__text">
+										Filter
+									</span>
+								</span>
+								<span class="fd-token fd-token--readonly">
+									<span class="fd-token__text">
+										Lorem
+									</span>
+								</span>
+								<span class="fd-tokenizer__indicator">
+									<div class="fd-popover">
+										<div class="fd-popover__control fd-input-group__control" aria-controls="F4GF5348a" aria-expanded="false" aria-haspopup="true">
+											<span class="fd-link">
+												4 more
+											</span>
+										</div>
+										<div class="fd-popover__body" aria-hidden="true" id="F4GF5348a">
+										</div>
+									</div>
+								</span>
+								<input class="fd-input fd-input-group__input fd-tokenizer__input" readonly aria-readonly="true" />
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+			</div>
+			<div class="fd-col fd-col--6">
+				<div class="fd-popover">
+					<div class="fd-popover__control" aria-expanded="false" aria-haspopup="true" aria-disabled="true" disabled>
+						<div class="fd-input-group fd-input-group--control"  aria-disabled="true" disabled>
+							<div class="fd-tokenizer">
+								<div class="fd-tokenizer__inner">
+									<span class="fd-token fd-token--readonly" role="button">
+										<span class="fd-token__text">
+											Bibendum
+										</span>
+									</span>
+									<span class="fd-token fd-token--readonly" role="button">
+										<span class="fd-token__text">
+											Lorem
+										</span>
+									</span>
+									<span class="fd-token fd-token--readonly" role="button">
+										<span class="fd-token__text">
+											Dolor
+										</span>
+									</span>
+									<span class="fd-token fd-token--readonly" role="button">
+										<span class="fd-token__text">
+											Filter
+										</span>
+									</span>
+									<span class="fd-token fd-token--readonly">
+										<span class="fd-token__text">
+											Lorem
+										</span>
+									</span>
+									<span class="fd-tokenizer__indicator">
+										<div class="fd-popover">
+											<div class="fd-popover__control fd-input-group__control" aria-controls="F4GcX34Xa" aria-expanded="false" aria-haspopup="true">
+												<span class="fd-link">
```

Comment:
Needs to be a button as suggested above.
```suggestion
												<span class="fd-tokenizer__indicator" role="button>
```